client.go

```diff
@@ -183,7 +183,7 @@ func echo(client *rpc2.Client, args []interface{}, reply *[]interface{}) error {
 
 // RFC 7047 : Update Notification Section 4.1.6
 // Processing "params": [<json-value>, <table-updates>]
-func update(client *rpc2.Client, params []interface{}, reply *interface{}) error {
+func update(client *rpc2.Client, params []interface{}, _ *interface{}) error {
```

Comment:
does it really needed to have func with ignored arg?

Reply:
the `github.com/cenkalti/rpc2` module that calls the `update` callback function on receiving the `update` method from the OVSDB-Server expects all the callback function to have two arguments. That library doesn't know that the `update` is one-way call and doesn't expect reply.

With `reply` not changed to `_`, it is getting tagged as error in my ide, so the common convention that I have seen is to set it '_'.